### ISSUE TITLE
panel.border removes axis lines

### ISSUE THREAD

**Issue by fromdputto** [hadley/ggplot2#1580]
using 

> theme(panel.border = element_blank()) 

removes the x and y axis lines from the plot and plot is left without any axis lines!!

```
df <- data.frame(x=seq(1:100), y=seq(1:100))
plot = ggplot(df, aes(x, y)) + geom_point() 
plot+ theme_bw() +theme(panel.border = element_blank())
```

**Comment by Katiedaisey** [comment]
This is related to #1565 and #1567. It should be fixed shortly by #1581. Until then you can use the work around 
`plot + theme_bw() + theme(panel.border = element_blank(), axis.line.x = element_line(), axis.line.y =         element_line())`

**Comment by hadley** [comment]
Closing because it's a duplicate.
